code/game/objects/items/devices/traitordevices.dm

```diff
@@ -223,3 +223,85 @@ effective or pretty fucking useless.
 		else
 			charge = min(max_charge,charge + 50) //Charge in the dark
 		animate(user,alpha = Clamp(255 - charge,0,255),time = 10)
+
+
+/* Research Director Syndie teleporter */
+/obj/item/device/experimental_teleporter
+	name = "experimental teleporter"
+	desc = "An imperfect teleportation device."
+	icon = 'icons/obj/device.dmi'
```

Comment:
[Please put all new icons under the hippie module](https://github.com/HippieStation/HippieStation13/blob/master/icons/hippie/README.txt)